curriculum/challenges/english/25-front-end-development/quiz-web-standards/66f1a0fcab4dfe9fcdcb269d.md

```diff
@@ -17,439 +17,439 @@ Answer all of the questions below correctly to pass the quiz.
 
-Placeholder question
+What does the TCP three-way handshake process achieve in web communication?
 
 #### --distractors--
 
-Placeholder distractor 1
+Establishes a secure communication channel
 
 ---
 
-Placeholder distractor 2
+Initiates and terminates an HTTP request
 
 ---
 
-Placeholder distractor 3
+Encodes the IP address for DNS resolution
 
 #### --answer--
 
-Placeholder answer
+Confirms the connection between client and server for reliable data transfer
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the one disadvantage of using dynamic content over static content on a website?
 
 #### --distractors--
 
-Placeholder distractor 1
+Dynamic content loads faster than static content
 
 ---
 
-Placeholder distractor 2
+Static content cannot be updated without modifying server-side code
 
 ---
 
-Placeholder distractor 3
+Dynamic content does not require server-side languages
 
 #### --answer--
 
-Placeholder answer
+Dynamic content is difficult to cache effectively
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is a key principle of web standards?
 
 #### --distractors--
 
-Placeholder distractor 1
+Platform dependency
 
 ---
 
-Placeholder distractor 2
+High vendor lock-in
 
 ---
 
-Placeholder distractor 3
+Proprietary formats
 
 #### --answer--
 
-Placeholder answer
+Backward compatibility
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which of the following is not a type of asset returned in an HTTP response?
 
 #### --distractors--
 
-Placeholder distractor 1
+HTML file
 
 ---
 
-Placeholder distractor 2
+CSS file
 
 ---
 
-Placeholder distractor 3
+JavaScript file
 
 #### --answer--
 
-Placeholder answer
+SSH key
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the primary advantage of dynamic files over static files?
 
 #### --distractors--
 
-Placeholder distractor 1
+Faster load times
 
 ---
 
-Placeholder distractor 2
+No server-side processing required
 
 ---
 
-Placeholder distractor 3
+More cacheable by browsers
 
 #### --answer--
 
-Placeholder answer
+Ability to generate content based on user input
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+What is the main benefit of following web standards during development?
 
 #### --distractors--
 
-Placeholder distractor 1
+Higher performance on proprietary browsers
 
 ---
 
-Placeholder distractor 2
+Allows for vendor-specific features
 
 ---
 
-Placeholder distractor 3
+Excludes older devices from support
 
 #### --answer--
 
-Placeholder answer
+Ensures cross-browser compatibility
 
 ### --question--
 
 #### --text--
 
-Placeholder question
+Which phase of the web standards lifecycle comes after 'Working Draft'?
 
 #### --distractors--
 
-Placeholder distractor 1
+Public implementation
 
 ---
 
-Placeholder distractor 2
+Editor’s Draft
 
 ---
 
-Placeholder distractor 3
+Final Recommendation
 
 #### --answer--
 
-Placeholder answer
+Candidate Recommendation
```

Comment:
Just as an aside, Candidate Recommendation is in the process of being changed.

Source: https://github.com/w3c/process/issues/402 

But for now it works.

Reply:
Thanks! I appreciate the heads-up about the changes.